utils/unittest/block.go

```diff
@@ -1,9 +1,14 @@
 package unittest
 
 import (
+	"fmt"
+
 	"github.com/onflow/flow-go/model/flow"
 )
 
+const GenesisProtocolStateIDHex = "4745d52bae83d7e106865a57617a2d3310403fce45ff21a8786aa47c5bd047c5"
+const GenesisParentIDHex = "037e61db6167dd01dd0dcbaf6a5c3656e87083672e9d8862f4e1af9d34bc98e7"
```

Comment:
I think of genesis as the very first block of a network, specifically excluding the root blocks of subsequent sporks. Genesis blocks by that definition always have no parent block ID.